.buildkite/integration.pipeline.yml

```diff
@@ -20,6 +20,19 @@ steps:
           provider: "gcp"
           machineType: "n2-standard-8"
 
+      - label: "Packaging: Ubuntu x86_64 FIPS"
+        key: "packaging-ubuntu-x86-64-fips"
+        env:
+          PACKAGES: "tar.gz"
+          PLATFORMS: "linux/amd64"
+          FIPS: "true"
+        command: ".buildkite/scripts/steps/integration-package.sh"
+        artifact_paths:
+        - build/distributions/**
+        agents:
+          provider: "gcp"
+          machineType: "n2-standard-8"
```

Comment:
same question regarding the size, do we need 30Gi or is a machine with ~16Gi line `n2-standard-4` enough?